assets/prototype/application/utilities/money/amountsMatch.ts

```diff
@@ -0,0 +1,14 @@
+import parsedAmount from './parsedAmount';
+
+/**
+ * returns true if the two supplied values are equal after being parsed as floats
+ *
+ * @param {number|string} amount1
+ * @param {number|string} amount2
+ * @return {boolean}
+ */
+const amountsMatch = (amount1: number | string, amount2: number | string): boolean => {
```

Comment:
I noticed in `date-fns`, the use instead of arg1/ arg2 the `dateLeft, dateRight` approach: https://date-fns.org/v2.9.0/docs/compareAsc
may not be relevant here, but worth mentioning

Reply:
ya I think that makes more sense in the context of a sorting algorithm like they are using that function for, where you are stepping through a list and comparing a list item against the one before it. This function is just for comparing any two amounts that don't even have to be from the same "container", so deciding which is the "left" item and which is the "right" item doesn't really make sense imho.

Thank you for bringing that up though because it's always good to be looking at how things are done in other places. We don't want to work in a bubble.